mantidproject/mantid
# Indirect - Data Analysis Interfaces - Documentation Update

This PR updates the documentation for the Indirect Data Analysis interfaces, providing up-to-date information on the new features in the QENS fitting interfaces (i.e. multiple data selection, selection of discontinuous spectra ranges and masking ranges). The additional feature for selecting the number of iterations in the monte-carlo routine for error calculations in I(Q,t) has also been described in the documentation.

This PR also removes out-of-date information, such as references to FuryFit.

**To test:**
Ensure the documentation provides up-to-date information of each relevant QENS fitting interface (MSD Fit, I(Q,t) Fit, Conv Fit, F(Q) Fit) as well as I(Q,t). Ensure out-of-date information has been removed and the formatting of the documentation appears reasonable.



Fixes #22966. 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
